src/grammar.lalrpop

```diff
@@ -480,7 +480,7 @@ IntLiteral: i64 = {
 Int: UntypedExpr = {
     <s:@L> <value:IntLiteral> <e:@L> => UntypedExpr::Int {
         location: location(s, e),
-        value,
+        value: value.to_string(),
```

Comment:
Rather than calling `.to_string()` here I think it would be better to change `IntLiteral` to return a `String`, not an `Int`, that way we're not converting `String -> Int -> String`.

Reply:
Yep, that's better. Fixed